---
Requested Tests:
- test_tc_pair_unpair
Type: TestNameList
...
---
Begin Time: 1710536636301
Details: 'wrapper() missing 1 required positional argument: ''self'''
End Time: 1710536645009
Extra Errors: {}
Extras: null
Result: ERROR
Retry Parent: null
Signature: test_tc_pair_unpair-1710536636301
Stacktrace: "Traceback (most recent call last):\n  File \"/Users/chaitanyajandhyala/code/python/template_code/Functional/.venv/lib/python3.9/site-packages/mobly/base_test.py\"\
    , line 783, in exec_one_test\n    test_method()\n  File \"/Users/chaitanyajandhyala/code/python/template_code/Functional/matter_qa/library/helper_libs/matter_testing_support.py\"\
    , line 1111, in async_runner\n    return asyncio.run(body(*args, **kwargs))\n\
    \  File \"/AppleInternal/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/asyncio/runners.py\"\
    , line 44, in run\n    return loop.run_until_complete(main)\n  File \"/AppleInternal/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/asyncio/base_events.py\"\
    , line 642, in run_until_complete\n    return future.result()\n  File \"/Users/chaitanyajandhyala/code/python/template_code/Functional/matter_qa/scripts/reliability_scripts/TC_Pair.py\"\
    , line 51, in test_tc_pair_unpair\n    tc_pair_unpair()\nTypeError: wrapper()\
    \ missing 1 required positional argument: 'self'\n"
Termination Signal Type: TypeError
Test Class: TC_Pair
Test Name: test_tc_pair_unpair
Type: Record
UID: null
...
---
Error: 1
Executed: 1
Failed: 0
Passed: 0
Requested: 1
Skipped: 0
Type: Summary
...
